### PR TITLE
MergeSort file

### PR DESCRIPTION
**Pull request by iamkunalpitale**
### **Describe your change:**

- [ ] Add an algorithm?Yes
- [ ] Fix a bug or typo in an existing algorithm? No
- [ ] Documentation change? No

### CODE REVIEW

File: src/main/java/com/thealgorithms/misc/MergeSort.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ 
+        while (j < high)                           //merge the remaining elements from right subarray
+        {
+            arr[k] = R[j];
+            j++;
+            k++;
+        }
+    }
+ 
+
```
Comment (Kavyapaidi) [inline src/main/java/com/thealgorithms/misc/MergeSort.java:57]: helper function that creates the sub cases for sorting

Comment (Kavyapaidi) [inline src/main/java/com/thealgorithms/misc/MergeSort.java:57]: helper function that creates the sub cases for sorting

**Comment by Kavyapaidi** [comment]
Merging can be performed automatically with 1 approving review.

File: src/main/java/com/thealgorithms/misc/MergeSort.java (hunk 1 above)
Comment (iamkunalpitale) [reply src/main/java/com/thealgorithms/misc/MergeSort.java:57]: What need to change here?

**Review by iamkunalpitale — COMMENTED** [review]
Remove spaces

**Comment by siriak** [comment]
Mergesort is already implemented in sorts module

**Pull request closed without merging**